src/main/java/com/gamingmesh/jobs/PermissionManager.java

```diff
@@ -119,41 +119,41 @@ public Double getMaxPermission(JobsPlayer jPlayer, String perm) {
 	return getMaxPermission(jPlayer, perm, false);
     }
 
-    public Double getMaxPermission(JobsPlayer jPlayer, String perm, boolean force) {
-	if (jPlayer == null || jPlayer.getPlayer() == null)
-	    return 0D;
+	public Double getMaxPermission(JobsPlayer jPlayer, String perm, boolean force) {
+		if (jPlayer == null || jPlayer.getPlayer() == null)
+			return 0D;
 
-	perm = perm.toLowerCase();
-	if (!perm.endsWith("."))
-	    perm += ".";
+		perm = perm.toLowerCase();
+		if (!perm.endsWith("."))
+			perm += ".";
 
-	HashMap<String, Boolean> permissions = jPlayer.getPermissionsCache();
-	if (force || permissions == null || getDelay(perm) + jPlayer.getLastPermissionUpdate() < System.currentTimeMillis()) {
-	    permissions = getAll(jPlayer.getPlayer());
-	    jPlayer.setPermissionsCache(permissions);
-	    jPlayer.setLastPermissionUpdate(System.currentTimeMillis());
-	}
+		HashMap<String, Boolean> permissions = jPlayer.getPermissionsCache();
+		if (force || permissions == null || getDelay(perm) + jPlayer.getLastPermissionUpdate() < System.currentTimeMillis()) {
+			permissions = getAll(jPlayer.getPlayer());
+			jPlayer.setPermissionsCache(permissions);
+			jPlayer.setLastPermissionUpdate(System.currentTimeMillis());
+		}
 
-	if (permissions == null) {
-	    return 0D;
-	}
+		if (permissions == null) {
+			return 0D;
+		}
 
-	Double amount = 0D;
-	for (String uno : permissions.keySet()) {
-	    if (uno.startsWith(perm)) {
-		double t = 0d;
-		try {
-		    t = Double.parseDouble(uno.replace(perm, ""));
-		} catch (NumberFormatException e) {
+		Double amount = 0D;
+		for (String uno : permissions.keySet()) {
+			if (uno.startsWith(perm)) {
+				double t = 0d;
+				try {
+					t = Double.parseDouble(uno.replace(perm, ""));
+				} catch (NumberFormatException e) {
+				}
+				if (amount < t)
+					amount = t;
+			}
 		}
 
-		amount += t;
-	    }
+		return amount;

```

Comment:
Literally this changes breaks the ability to sum up the boosts with permissions, see https://github.com/Zrips/Jobs/commit/f61fbfeea84d466609c827ea3fefbcd66fc08d48

Reply:
Sorry, I'm new to the fork of github. Didn't think you would get the pull request. I thought it was just in my repo.

There is something I don't get then. Maybe you can help me. I have a jobs.max.2 at a certain level. Levels inherit from each other. The next level is jobs.max.3. I don't want to sum them. I want the new max. the problem is that the jobs.max.2 is still there. So it's like it add jobs.max.2 and jobs.max.3 and they get 5 jobs instead of 3. I use LuckPerms.

Thanks for the reply.

Reply:
Thats why I closed this PR. I committed a fix, so now it's been fixed.